Formula/liblerc.rb

```diff
@@ -0,0 +1,32 @@
+class Liblerc < Formula
+  desc "Ersi LERC library (Limited Error Raster Compression)"
+  homepage "https://github.com/Esri/lerc"
+  url "https://github.com/Esri/lerc/archive/refs/tags/v4.0.0.tar.gz"
+  sha256 "91431c2b16d0e3de6cbaea188603359f87caed08259a645fd5a3805784ee30a0"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    (testpath/"test.cc").write <<~EOS
+      #include <Lerc_c_api.h>
+      #include <Lerc_types.h>
+      int main() {
+      }
+    EOS
+
+    system ENV.cxx, "test.cc", "-std=c++11",
+                    "-I#{include}",
+                    "-L#{lib}",
+                    "-lLerc",
+                    "-o", "test_liblerc"
+
+    assert_equal("", shell_output("./test_liblerc"))
```

Comment:
Improve the test a bit
```suggestion
    (testpath/"test.cc").write <<~EOS
      #include <Lerc_c_api.h>
      #include <Lerc_types.h>
      int main() {
        const int infoArrSize = (int)LercNS::InfoArrOrder::_last;
        const int dataRangeArrSize = (int)LercNS::DataRangeArrOrder::_last;
        lerc_status hr(0);

        return 0 ;
      }
    EOS

    system ENV.cxx, "test.cc", "-std=gnu++17",
                    "-I#{include}",
                    "-L#{lib}",
                    "-lLerc",
                    "-o", "test_liblerc"

    assert_empty shell_output("./test_liblerc")
```